README.md

```diff
@@ -1,8 +1,8 @@
-# Jitsuin RKVST Documentation
+# RKVST Documentation
 
-This is the Home of the Jitsuin RKVST Documentation
+This is the Home of the RKVST Documentation
 
-Jitsuin RKVST is a Multi-Party Automation API powered by the Blockchain find out more on our [website](https://jitsuin.com) or read through our [Docs]()
+RKVST provides Continuous Assurance as a Service to attest trustworthiness of assets, and boost confidence in critical decisions.  Find out more on our [website](https://jitsuin.com).
```

Comment:
change the website link to RKVST.com

Reply:
That currently points at docs.jitsuin.com?

Is rkvst.com live yet?

Reply:
This isn't updated yet so we'll leave it

Reply:
imminent.